File: Java/alg09.java
import java.util.Scanner;
public class ALG09 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int X = scanner.nextInt();
        int Y;
        if (X >= 10)
            Y = (int) Math.pow(X, 2);
        else
            Y = (int) Math.pow(X, 3);
        System.out.println(Y);
        scanner.close();
    }
}

File: C#/alg09.cs
using System;
class ALG09
{
    static void Main()
    {
        int X;
        int Y;
        int.TryParse(Console.ReadLine(), out X);
        if (X >= 10)
            Y = (int)Math.Pow(X, 2);
        else
            Y = (int)Math.Pow(X, 3);
        Console.WriteLine(Y);
    }
}
